### DISCUSSION TITLE
Daily Scrum Meeting #1

### DISCUSSION THREAD

**Discussion by IrfanZnA** [GIA-003/Kado#15] [General]
mereview hasil pekerjaan hari ini

**Comment by Rizkiblack** [comment]
saya sudah mentranskrip hasil wawancara dan tidak ada kendala

**Comment by IrfanZnA** [comment]
trus apa lagi?

**Reply by naufalfadhlullah** [reply]
Aku sudah membuat sketsa tanpa hambatan

**Comment by IrfanZnA** [comment]
trus apa lagi?

**Reply by HudaGates** [reply]
Aku sudah membeli kado namun kendala hujan jadi untuk pita kucari dihari berikutnya dan semua ini tinggal didekor aja

**Reply by IrfanZnA** [reply]
oke gapapa

**Comment by IrfanZnA** [comment]
ini udah clear semua kan?

**Reply by naufalfadhlullah** [reply]
Aman ngab

**Reply by HudaGates** [reply]
A1 Aman

**Comment by IrfanZnA** [comment]
ku close ya

**Reply by naufalfadhlullah** [reply]
Yoi

**Reply by HudaGates** [reply]
Emoh
